libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -145,10 +173,34 @@ export class RadioGroupComponent implements AfterContentInit {
     if (value === this._value) return;
     this.setSelectedItem(value);
     this.valueChange.emit(value);
+
+    if (this.boundToForm) {
+      this._onChangeCallback(value);
+    }
   }
+
   // #endregion "protected" methods used by template
 
   // #region private methods
+
+  private trackTouchedState() {
```

Comment:
I haven't looked into this yet, so the commented out methods here serves as a reminder-to-self